src/pages/hegel/reference/becoming/sublation.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Sublation
+seoTitle: Sublation in Hegel's Logic
+description: Learn about sublation from Hegel's philosophy.
+isArticle: true
+authors: Filip Niklas (2024)
+editors:
+contributors:
+---
+
+## Sublation
+
+> _To sublate_ and _being sublated_ (the _idealized_) constitute one of the most
+> important concepts of philosophy (Hegel 2010, 81/21.94).
+
+The importance of the concept of sublation [_Aufhebung_] in Hegel's philosophy
+cannot be understated. It is a notion that is virtually omnipresent in Hegel's
+entire systematic oeuvre and plays a chief role in the dialectical determination
+of ideas and concepts.
+
+However, despite its pivotal significance, it does not receive direct treatment
+in _The Science of Logic_, Hegel's major work on logic and metaphysics, apart
+from a remark that follows the
+[`development of becoming`](/hegel/reference/becoming/development). The remark
+in question, however, is instructive given its context and illustrates the
+essential three-pronged structure of sublation.
+
+While sublation enjoys common usage in the German language, there is no direct
+equivalent in English. And so sublation has become something of a highly
+technical term that is most often used in Hegelian scholarship. This is
+unfortunate, since the speculative element in the word itself never natively
+manifests to the thinker, and one must continuously bring together disparate
+meanings into this rather awkward term.
+
+### Semantics
+
+The German _aufheben_ has a twofold meaning, Hegel writes. On the one hand, it
+means to _to keep_ or _to preserve_. On the other hand, it means _to cause to
+cease_ or _to put an end to_.
+
+But to preserve something, Hegel notes, has a negative tinge to it because to
+preserve something one must remove it from its immediacy in order to _retain_
```

Comment:
I don't think one can "remove something from its immediacy" - what about "undermine its immediacy" or "stop it from being immediate"? I think my issue is with the verb "remove" - you can remove something/one from a situation but you cannot remove them from a state. I cannot remove you from anger or from sleep, etc.

Reply:
Good call, implemented "undermine" 👍